docs/reference/config.md

```diff
@@ -30,7 +30,7 @@ nav_order: 2
 
 |Key|Description|Type|Default Value|
 |---|-----------|----|-------------|
-|keyNormalization|Mechanism to normalize keys before using them. Valid options are `blockchain_plugin` - use blockchain plugin (default) or `none` - do not attempt normalization|`string`|`<nil>`
+|keynormalization|Mechanism to normalize keys before using them. Valid options are `blockchain_plugin` - use blockchain plugin (default) or `none` - do not attempt normalization (deprecated - namespace.predefined[].asset.manager.keynormalization)|`string`|`<nil>`
```

Comment:
`namespaces` not `namespace`
and for clarity might want to say "_use_ namespaces.predefined[].asset.manager.keynormalization"